xbmc/GUIInfoManager.cpp

```diff
@@ -4675,6 +4675,11 @@ const infomap pvr[] =            {{ "isrecording",              PVR_IS_RECORDING
 /// @{
 /// \table_start
 ///   \table_h3{ Labels, Type, Description }
+///   \table_row3{   <b>`PVR.EpgEventIcon`</b>,
+///                  \anchor PVR_EpgEventIcon
+///                  _string_,
+///     Returns the thumbnail for the epg event associated with the item (if it exists)
```

Comment:
Please change the text to reflect that this returns the icon of the currently playing epg event, if any - like phrased in the other comments. => "Returns the icon of the currently playing epg event, if any."